.cookiecutter/cookiecutter.json

```diff
@@ -1,7 +1,7 @@
 {
   "template": "https://github.com/hypothesis/cookiecutters",
   "directory": "pypackage",
-  "ignore": [],
+  "ignore": ["tests/factories/*"],
```

Comment:
`test-pypackage` does have a DB (just in order to test the cookiecutter's ability to put all the DB boilerplate stuff into packages as well as apps) but it doesn't actually have a DB session or any models or factories so I didn't think the cookiecutter's factories stuff (the utility function for hooking up `factory_boy` to an `sqlalchemy` session) was relevant to `test-pypackage`. Just ignore it here and then the cookiecutter won't make any changes to `test-pypackage`